### ISSUE TITLE
<style> tag may not get processed as expected in all scenarios

### ISSUE THREAD

**Issue by BenParizek** [barrelstrength/craft-sprout-email#146]
Review comments in ticket:

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_1ge2r)

**Comment by johnj24** [comment]
I can't see the comments in the link, but I'm experiencing issues with the <style> tag as well. I've tested to the point where emails are not sent if you have more than one class in your style. And even if you only have just the one style, I'm not sure it's even applied (at least not in my Gmail account). I set up `body { background: black; }` and while it turned the body black in the notification preview, it didn't have any effect on the email body in the Gmail web client.

For example, I've noticed that this sends (whether or not it's rendered in the email client, I can't tell):
```
<style type="text/css">
  .u-p-4 { padding: 1rem; }
</style>
```
but this doesn't...
```
<style type="text/css">
  .u-p-4 { padding: 1rem; }
  .u-mb-4 { margin-bottom: 1rem; }
</style>
```
Just by adding that second style, the e-mail never gets sent. The Sprout Forms test e-mail modal just disappears with no feedback.
